src/components/postdetail/QnASection.tsx

```diff
@@ -0,0 +1,34 @@
+import styles from './qnaSection.module.scss';
+
+const QnASection = () => {
+  return (
+    <div className={styles.container}>
+      <h1 className={styles.title}>담당자 Q&A</h1>
+      <div className={styles.divider}></div>
+      <div className={styles.inputContainer}>
+        <textarea
+          className={styles.textArea}
+          placeholder=''></textarea>
```

Comment:
태그 안에 내용이 아무것도 없다면 바로 닫는게 편해보여요!

```suggestion
        <textarea
          className={styles.textArea}
          placeholder='' />
```